Support update faq (#96)
stoplightio/platform-docs: c.troubleshooting.md

# FAQ

Below you will find answers to commonly encountered questions.

> If you are having trouble and cannot find a suitable answer, don't hesitate to [let us know](mailto:support@stoplight.io).

## Why is there an "Others" section in the sidebar?

If you're seeing an "Others" section in the sidebar of your project documentation...

![Others](assets/images/others_missing_tag.png)

Grouping of endpoints and models is done with [tags](https://swagger.io/docs/specification/grouping-operations-with-tags/). To prevent an "Others" section from being generated, be sure to add at least one tag to each endpoint and model in your project.

You can find more information on the default order of the project sidebar (as well as how to customize it) [here](https://meta.stoplight.io/docs/platform/4.-documentation/d.table-of-contents.md).

## How do I export a file from Stoplight?

> If you need to retrieve a file published in Stoplight, the quickest way to do so is using the **Export** buttons found in both Explore and your project documentation.

Reviewer: Are we calling it Explore or Explorer? We say both a few lines apart but it’s nice to be consistent whenever possible.

> If you need to retrieve a file published in Stoplight, the quickest way to do so is using the **Export** buttons found in both Explore and your project documentation.

Reviewer 2: ```suggestion
If you need to retrieve a file published in Stoplight, the quickest way to do so is using the **Export** buttons found in both Explorer and your project documentation.
```

***Exporting using Explorer***
![Export through Explorer](assets/images/export_explorer.png)

***Exporting within Studio***
![Export through Studio](assets/images/export_studio.png)

### What are the differences between Original, Bundled, Dereferenced.
1. Original - is the raw file keeping the $refs
2. Bundled - resolves remote $refs once, re-referencing the same objects in subsequent references (producing a smaller file)
3. Dereferenced - resolves all $refs, inserting each resolved reference inline

## Does Stoplight support adding HTML to documentation?

> HTML support within Markdown is limited. See our [Markdown documentation](https://meta.stoplight.io/docs/studio-demo/docs/markdown/basic-syntax.md) for more information.

Reviewer: Would this link be more appropriate? https://meta.stoplight.io/docs/studio/docs/Documentation/03-markdown-basics.md

Reviewer 2: ```suggestion
HTML support within Markdown is limited. See our [Markdown documentation](https://meta.stoplight.io/docs/studio/docs/Documentation/03-markdown-basics.md) for more information.
```

Reviewer 2: Good point. Updated.

# FAQ

Below you will find answers to commonly encountered questions.

> If you are having trouble and cannot find a suitable answer, don't hesitate to [let us know](mailto:support@stoplight.io).

## Why is there an "Others" section in the sidebar?

If you're seeing an "Others" section in the sidebar of your project documentation...

![Others](assets/images/others_missing_tag.png)

> - Tags will need to be added to the models and/or endpoints to prevent them from being placed into the Others section of the Table of Contents
> - View [Default Ordering](https://meta.stoplight.io/docs/platform/4.-documentation/d.table-of-contents.md#the-default-structure) of the table of contents

Reviewer 2: ```suggestion
Grouping of endpoints and models is done with [tags](https://swagger.io/docs/specification/grouping-operations-with-tags/). To prevent an "Others" section from being generated, be sure to add at least one tag to each endpoint and model in your project.

You can find more information on the default order of the project sidebar (as well as how to customize it) [here](https://meta.stoplight.io/docs/platform/4.-documentation/d.table-of-contents.md).
```

## How do I export a file from Stoplight?

If you need to retrieve a file published in Stoplight, the quickest way to do so is using the **Export** buttons found in both Explore and your project documentation.

***Exporting using Explorer***
![Export through Explorer](assets/images/export_explorer.png)

***Exporting within Studio***
![Export through Studio](assets/images/export_studio.png)

### What are the differences between Original, Bundled, Dereferenced.
1. Original - is the raw file keeping the $refs
2. Bundled - only resolves remote $refs 

> 3. Dereferenced- resolves all remote and inline $refs

Reviewer 2: ```suggestion
3. Dereferenced - resolves all $refs, inserting each resolved reference inline
```

## Does Stoplight support adding HTML to documentation?

> HTML support within Markdown is limited.
> 
> See our [Markdown Documentation](https://meta.stoplight.io/docs/studio-demo/docs/markdown/basic-syntax.md) for more information.

Reviewer 2: ```suggestion
HTML support within Markdown is limited. See our [Markdown documentation](https://meta.stoplight.io/docs/studio-demo/docs/markdown/basic-syntax.md) for more information.
```
